File: Course_01/Java Language/Matrizes/soma_linhas.java
package Matrizes;

import java.util.Locale;
import java.util.Scanner;

public class soma_linhas {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int i, j, m, n;
		
		System.out.print("Qual a quantidade de linhas da matriz? ");
		m = sc.nextInt();
		System.out.print("Qual a quantidade de colunas da matriz? ");
		n = sc.nextInt();
		
		double [][] matriz = new double [m][n];
		
		for (i = 0; i < m; i++) {
			System.out.println("Digite os elementos da " + (i + 1) + "a. linha:");
			for (j = 0; j < n; j++) {
				matriz[i][j] = sc.nextDouble();
			}
		}
		
		double [] vetor = new double [m];
		
		for (i = 0; i < m; i++) {
			vetor[i] = 0;
			for (j = 0; j < n; j++) {
				vetor[i] += matriz[i][j];
			}
		}
		
		System.out.println("VETOR GERADO:");
		
		for (i = 0; i < m; i++) {
			System.out.print(vetor[i] + " ");
		}
				
				
		sc.close();
	}

}

File: Course_01/Python/Matrizes/soma_linhas.py
m = int(input("Qual a quantidade de linhas da matriz? "))
n = int(input("Qual a quantidade de colunas da matriz? "))

matriz = [[0 for x in range(n)] for x in range(m)]

for i in range(0, m):
    print(f"Digite os elementos da {i+1}a. linha:")
    for j in range(0, n):
        matriz[i][j] = float(input())

vetor = [0 for x in range(m)]
soma = 0
for i in range(0, m):
    for j in range(0, n):
        soma += matriz[i][j]
    vetor[i] = soma
    soma = 0
print("Vetor gerado:")
for i in range(0, m):
    print(vetor[i])